book/09-git-and-other-scms/sections/import-svn.asc

```diff
@@ -91,6 +91,14 @@ $ rm -Rf .git/refs/remotes
 ----
 
 Now all the old branches are real Git branches and all the old tags are real Git tags.
+
+There's one last thing to clean up. Unfortunately, git svn creates an extra branch named trunk, which maps to Subversion's default branch, but the trunk ref points to the same place as master. Since master is more idiomatically Git, here's how to remove the extra branch:
```

Comment:
One last thing: can you set some of the "trunk"s and "master"s in `monospace`? Maybe like this:

```markdown
There's one last thing to clean up. Unfortunately, `git svn` creates an extra 
branch named `trunk`, which maps to Subversion's default branch, but the 
`trunk` ref points to the same place as `master`. Since `master` is more
idiomatically Git, here's how to remove the extra branch:
```